src/workload.rs

```diff
@@ -329,6 +329,8 @@ impl WorkloadInformation {
 #[derive(serde::Serialize, Default, Debug)]
 pub struct WorkloadStore {
     workloads: HashMap<IpAddr, Workload>,
+    // workload_to_vip maintains a mapping of workload IP to VIP. This is used only to handle removals.
+    workload_to_vip: HashMap<IpAddr, HashSet<(SocketAddr, u16)>>,
```

Comment:
I think we can record the vips in `Workload`, then can get ride of this map

Reply:
We need a way to efficiently lookup a VIP though